wiki/Tournaments/GO/3/en.md

```diff
@@ -73,13 +75,184 @@ The #Game\_Osu Tourney 3 is run by various community members.
 
 ## Participants
 
-*The participants will be revealed after the registration phase closes.*
+| BWS Rank |  | Player |
```

Comment:
is there a wiki page about BWS? this would make no sense to most people

Reply:
Not an official article, but there's a link in the ruleset that explains it.
Or should i import the short definition to the Participants section?

Reply:
yeah just a short description is enough. maybe we can give it its own article sometime and have that description link to it in the future

Reply:
```markdown
*Note: Badge-weighted seeding (BWS) is used in this tournament to more accurately seed players and enforce rank restrictions. It works by taking into account the amount of recorded tournament wins (via badges) that a player has in addition to their rank. See the Game\_Osu website linked above for more details.*
```
?